app/ide-desktop/lib/dashboard/src/authentication/src/components/app.tsx

```diff
@@ -38,12 +38,13 @@ import * as react from 'react'
 import * as router from 'react-router-dom'
 import * as toast from 'react-hot-toast'
 
-import * as projectManagerModule from 'enso-content/src/project_manager'
+import * as app from '../../../../../../../../target/ensogl-pack/linked-dist/index'
```

Comment:
i'm not sure unfortunately - not only is it outside of this module, it's even outside of the monorepo (?) root ( D: )

but also i don't think it can currently be added as a workspace submodule as it doesn't have a package.json

we *could* use typescript's `paths` option. it *is* supported out of the box by esbuild but is pretty fragile as it's not supported out of the box by most tools. also it just moves the `../../../../../` to elsewhere so i'm not sure it'd be an improvement, unfortunately

Reply:
oh actually after removing `ide?: app.App` this is no longer needed